cvat/apps/engine/signals.py

```diff
@@ -3,21 +3,17 @@
 # SPDX-License-Identifier: MIT
 import shutil
 
+from django.contrib.auth.models import User
 from django.db.models.signals import post_delete, post_save
 from django.dispatch import receiver
-from django.contrib.auth.models import User
 
-from .models import (
-    Data,
-    Job,
-    StatusChoice,
-    Task,
-    Profile,
-)
+from .models import (CloudStorage, Data, Job, Profile, Project,
+    StatusChoice, Task)
 
 
-@receiver(post_save, sender=Job, dispatch_uid="update_task_status")
-def update_task_status(instance, **kwargs):
+@receiver(post_save, sender=Job,
+    dispatch_uid=__name__ + ".update_task_status_handler")
+def __update_task_status_handler(instance, **kwargs):
```

Comment:
I would not recommend to use names beginning with two underscores, because such names can be subject to Python's name mangling feature. One underscore is sufficient.

Reply:
It will be even good if it will be mangled. The name isn't important. It should not be used anywhere in the code.